### PR TITLE
chore: update CODEOWNERS

### PR DESCRIPTION
**Pull request by Jolg42**
That should reduce a bit of noise in notifications, people can still review each other's PR of course but won't get pinged for everything

### CODE REVIEW

**Comment by Jolg42** [comment]
Note: We can merge without CI tests since it's just a .md file here

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -3,3 +3,14 @@
...
+# Schema Team
+/src/packages/migrate/           @Jolg42 @jkomyno
+
+# Client Team
+/src/packages/client/            @millsp @aqrln @SevInf @danstarns
+/src/packages/react-prisma/      @millsp @aqrln @SevInf @danstarns
+/src/packages/integration-tests/ @millsp @aqrln @SevInf @danstarns
+/src/packages/engine-core/       @millsp @aqrln @SevInf @danstarns
+/src/packages/generator-helper/  @millsp @aqrln @SevInf @danstarns
+/src/packages/get-platform/      @millsp @aqrln @SevInf @danstarns
```
Comment (aqrln) [inline .github/CODEOWNERS:16]: `get-platform` is in a different repo. Was this supposed to be something else?

Comment (Jolg42) [reply .github/CODEOWNERS:16]: Copy paste from the past, I heard it might come back here 😄 
I'll remove and when it comes back we can add it again

Comment (Jolg42) [inline .github/CODEOWNERS:16]:
```suggestion
```

**Review by aqrln — APPROVED** [review]

**Pull request merged**